pkg/k8shandler/fluentd.go

```diff
@@ -419,6 +439,13 @@ func newFluentdPodSpec(cluster *logging.ClusterLogging, trustedCABundleCM *v1.Co
 		{Name: "tmp", MountPath: "/tmp"},
 	}
 
+	exporterContainer.VolumeMounts = []v1.VolumeMount{
+		{Name: logVolumeMountName, MountPath: logVolumePath},
+		{Name: metricsVolumeName, MountPath: "/etc/fluent/metrics"},
+	}
+	// default parameters -->  -verbosity=2 -dir=/var/log/containers -http=:2112"
+	exporterContainer.Command = []string{"/bin/sh", "-c", "/usr/local/bin/log-file-metric-exporter  -verbosity=2  -dir=/var/log/containers -http=:2112 -keyFile=/etc/fluent/metrics/tls.key  -crtFile=/etc/fluent/metrics/tls.crt"}
```

Comment:
why do we need to wrapper with "/bin/sh" instead of simply calling /usr/local/bin/log-file-metric-exporter  and passing the args we need